src/PostgREST/Observation.hs

```diff
@@ -69,12 +67,12 @@ type ObservationHandler = Observation -> IO ()
 
 observationMessage :: Observation -> Text
 observationMessage = \case
-  AdminStartObs host port ->
-    "Admin server listening on " <> fromJust host <> ":" <> show (fromIntegral (fromJust port) :: Integer)
+  AdminStartObs address ->
+    "Admin server listening on " <> address
   AppStartObs ver ->
     "Starting PostgREST " <> T.decodeUtf8 ver <> "..."
-  AppServerPortObs host port ->
-    "API server listening on " <> host <> ":" <> show port
+  AppServerPortObs address ->
+    "API server listening on " <> address
   AppServerUnixObs sock ->
     "API server listening on unix socket " <> show sock
```

Comment:
If we can combine these two observations, then we can deduplicate the code even more in other places. WDYT?